canonical-web-and-design/maas-ui
# refactor(ui): make interface table columns reusable

## Done

- Make some columns from the interface table reusable for all node types.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit the network tab in machine details.
- The interfaces table should be unchanged.
- Open the edit form for an interface.
- The interfaces table should appear unchanged.

## Fixes

Fixes: canonical-web-and-design/app-tribe#570.
